SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentReader.cpp

```diff
@@ -317,14 +317,18 @@ void ComponentReader::ApplyProperty(Schema_Object* Object, Schema_FieldId FieldI
 				if (ObjectValue == nullptr)
 				{
 					// At this point, we're unable to resolve a stably-named actor by path. This likely means either the actor doesn't exist, or
-					// it's part of a streaming level that hasn't been streamed in. In either case, there's nothing we can do.
+					// it's part of a streaming level that hasn't been streamed in. In this case, queue the actor based on it's level.
 					FString FullPath;
 					improbable::GetFullPathFromUnrealObjectReference(ObjectRef, FullPath);
 					UE_LOG(LogSpatialComponentReader, Warning, TEXT("Object ref did not map to valid object, will be set to nullptr: %s %s"),
 						*ObjectRef.ToString(), FullPath.IsEmpty() ? TEXT("[NO PATH]") : *FullPath);
 					ObjectProperty->SetObjectPropertyValue(Data, nullptr);
+
+					InObjectReferencesMap.Add(Offset, FObjectReferences(ObjectRef, ParentIndex, Property));
+					UnresolvedRefs.Add(ObjectRef.GetLevelReference());
```

Comment:
Any way to differentiate between a stably named actor from level that wasn't loaded vs the actor doesn't exist? (the latter supposedly won't ever be resolved)